hassio/addons/validate.py

```diff
@@ -52,6 +52,7 @@ def check_network(data):
     vol.Optional(ATTR_PORTS): dict,
     vol.Optional(ATTR_HOST_NETWORK, default=False): vol.Boolean(),
     vol.Optional(ATTR_DEVICES): [vol.Match(r"^(.*):(.*):([rwm]{1,3})$")],
+    vol.Optional(ATTR_TMPFS): vol.Coerce(str)},
```

Comment:
There is a lot of possibility since it is mount option. I don't know how to cover them all. I can write one for my use case if needed (size and uid).